index.rst

```diff
@@ -146,6 +153,8 @@ RepositoryRef
 
 Globally unique, human parseable, identifier of a :ref:`Repository` (e.g. the path to it or a URI).
 
+[KSK] Do we want to to explicitly call out the term path?  A path could be URI given a filesystem.
+
```

Comment:
I'm not entirely fond of this concept as it stands right now in any case.  I think we need to better define the context in which a set of repositories can be labeled before trying to define the label.  In the OPS DBB, for instance, there might be  many repositories just identified by tags that live in the database, and there may be no global way to identify those relative to the repositories living on a local filesystem on someone's laptop.  @pschella and I talked a bit about this, but we only got as far as thinking we needed some concept for a group of repositories (which we're very provisionally calling a `RepositoryGroup`).